doc/isorms.xml

```diff
@@ -54,6 +54,68 @@
   </ManSection>
 <#/GAPDoc>
 
+<#GAPDoc Label="RMSIsoByTriple">
+  <ManSection>
+    <Oper Name = "RMSIsoByTriple" Arg = "R1, R2, triple"/>
+    <Oper Name = "RZMSIsoByTriple" Arg = "R1, R2, triple"/>
+    <Returns>An isomorphism.</Returns>
+    <Description>
+      If <A>R1</A> and <A>R2</A> are isomorphic Rees 0-matrix semigroups then
+      <C>RZMSIsoByTriple</C> returns the isomorphism between <A>R1</A> and
+      <A>R2</A> defined by <A>triple</A>, which should be a list consisting of
+      the following:
+
+      <List>
+        <Item>
+          <C><A>triple</A>[1]</C> should be a permutation describing an
+          isomorphism from the graph of <A>R1</A> to the graph of <A>R2</A>,
+          i.e. it should satisfy
+          <C>OnDigraphs(RZMSDigraph(<A>R1</A>), <A>triple</A>[1])
+             = RZMSDigraph(<A>R2</A>)</C>.
```

Comment:
Make it more explicit what this means for a Rees matrix semigroup without 0. I.e. that the associated graph of is always complete bipartite and so `triple[1]` should stabiliser `[1 .. NrColumns(R)]` setwise, and `[NrColumns(R) + 1 .. NrColumns(R) + NrRows(R)]`.